frdrpc/faraday.proto

```diff
@@ -360,11 +360,15 @@ message ExchangeRateResponse {
 }
 
 message BitcoinPrice {
-    // The price of 1 BTC, expressed in USD.
+    // The price of 1 BTC, expressed in the currency denoted in the currency
+    // field.
     string price = 1;
 
     // The timestamp for this price price provided.
     uint64 price_timestamp = 2;
+
+    // The currency that the price is denoted in.
+    string currency = 3;
```

Comment:
Wondering whether this should be an enum rather than a string? Perhaps not worth the effort, WDYT?

Reply:
would that not mean that we need to add an enum entry for each possible currency code option that a user could want to use? If it is a string then they can use any arbitrary currency. right?

Reply:
Oh yeah ofc, ignore!